compiler/natives/src/net/route/zsys_freebsd.go

```diff
@@ -0,0 +1,119 @@
+// Created by cgo -godefs - DO NOT EDIT
+// cgo -godefs defs_freebsd.go
+
+// +build js
```

Comment:
This line would make sense inside GOROOT, but it's not needed in this context, since this is a file inside:

```
gopherjs/compiler/natives/src/...
```

So it's clear this is meant for GopherJS.